### ISSUE TITLE
Tiny yolo

### ISSUE THREAD

**Issue by jayakrishna-a** [allanzelener/YAD2K#91]
Is tiny yolo overfits for 1 class problem? When I try to transfer learn tiny yolo for one class problem, the training loss is decreasing but the validation loss is not decreasing.
